crates/bevy_ecs/src/hierarchy.rs

```diff
@@ -0,0 +1,345 @@
+// TODO: REMOVE THIS
+#![allow(missing_docs)]
+
+//! Parent-Child relationships for entities.
```

Comment:
More docs on the "standard properties" of parent-child relationships would be good here.